docs/howto/UPGRADING.md

```diff
@@ -0,0 +1,93 @@
+# How to upgrade from previous versions
+
+### Creating a Custom Base Adapter
+
+You can upgrade to ams ``.10`` by creating a custom Base Adapter to
+emulate the response that your application currently serves.
+
+###### Base Adapter
+
+A base adapter takes a ``ActiveModel::Serializer`` or
+``ActiveModel::Serializer::ArraySerializer`` instance, and creates a hash
+used for serialization in its ``serializable_hash`` method. A base adapter
+could look like this:
+
+```ruby
+class MyApp::BaseAdapter < ActiveModel::Serializer::Adapter::Base
```

Comment:
Though after clicking comment, I realized the options definitions are wrong, should be:

```ruby
adapter = ActiveModelSerializers::V08::Adapter
serializer = ActiveModelSerializers::V08::ArraySerializer'
each_serializer = ActiveModelSerializers::V08::Serializer
```